qir/qat/ValidationPass/ValidationPassConfiguration.cpp

```diff
@@ -151,9 +194,23 @@ ValidationPassConfiguration ValidationPassConfiguration::fromProfileName(String
         profile.allow_internal_calls_     = false;
         profile.allowlist_external_calls_ = true;
         profile.allowlist_opcodes_        = true;
-        profile.opcodes_ =
-            OpcodeSet{{"call"}, {"ret"}, {"inttoptr"}, {"br"},   {"phi"}, {"add"},  {"sub"},    {"mul"},
-                      {"and"},  {"or"},  {"xor"},      {"lshr"}, {"shl"}, {"icmp"}, {"select"}, {"zext"}};
+        profile.opcodes_                  = OpcodeSet(
```

Comment:
Are the existing profile configs being kept for now for backward compat or do we expect to keep supporting these along with configs for the foreseeable future?

Reply:
Yes. They will be deprecated in time, but that needs to be coordinated with yourself and @cesarzc .

Reply:
We do have a larger refactor that needs to be made before we make said deprecation and also, if there is a strong wish to keep them, then we can consider it. It would, however, be in a different form than today.